TombEngine/Objects/TR2/Vehicles/skidoo.cpp

```diff
@@ -663,21 +666,20 @@ namespace TEN::Entities::Vehicles
 		FindNewTarget(*laraItem, weapon);
 		AimWeapon(*laraItem, lara->RightArm, weapon);
 
-		if (TrInput & VEHICLE_IN_FIRE && !skidooItem->ItemFlags[0])
+		if (IsHeld(In::DrawWeapon) && !skidooItem->ItemFlags[0])
 		{
 			auto angles = EulerAngles(
 				lara->RightArm.Orientation.x,
 				lara->RightArm.Orientation.y + laraItem->Pose.Orientation.y,
 				0
 			);
 
-			if ((int)FireWeapon(LaraWeaponType::Pistol, *lara->TargetEntity, *laraItem, angles) +
-				(int)FireWeapon(LaraWeaponType::Pistol, *lara->TargetEntity, *laraItem, angles))
-			{
-				skidoo->FlashTimer = 2;
-				SoundEffect(weapon.SampleNum, &laraItem->Pose);
-				skidooItem->ItemFlags[0] = 4;
-			}
+			FireWeapon(LaraWeaponType::Snowmobile, *lara->TargetEntity, *laraItem, angles);
+			FireWeapon(LaraWeaponType::Snowmobile, *lara->TargetEntity, *laraItem, angles);
+			//lara->LeftArm.GunFlash = 1;
```

Comment:
What are these two commented lines doing?

Reply:
It was a test for the snowmobile gunflash, but it's hardcoded, the flash was positioned at lara hand, not the snowmobile so i commented it for later.